google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionOptions.java

```diff
@@ -158,6 +159,7 @@ public String getDefaultValue() {
   private static final String DEFAULT_USER_AGENT = null;
   private static final String DEFAULT_OPTIMIZER_VERSION = "";
   private static final String DEFAULT_OPTIMIZER_STATISTICS_PACKAGE = "";
+  private static final RpcPriority DEFAULT_RPC_PRIORITY = RpcPriority.HIGH;
```

Comment:
Also here I would say that it might be better to use `null` (or some other 'undefined' value) as the default, and so prevent a priority from being included in every RPC invocation.